client/my-sites/domains/domain-management/transfer/transfer-to-other-user/transfer-domain-to-other-user.jsx

```diff
@@ -313,13 +313,13 @@ class TransferDomainToOtherUser extends Component {
 				<>
 					<p>
 						{ translate(
-							'Please choose an administrator to transfer domain mapping of {{strong}}%(domainName)s{{/strong}} to.',
+							'Please choose an administrator to transfer domain connection of {{strong}}%(domainName)s{{/strong}} to.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.